gato/github/api.py

```diff
@@ -6,11 +6,14 @@
 import zipfile
 import re
 import io
+import json #Jimmy
 
 from gato.cli import Output
 from datetime import datetime, timezone, timedelta
 
 logger = logging.getLogger(__name__)
+logging.root.setLevel(logging.DEBUG)
```

Comment:
```suggestion
```
We don't need to log at the debug level otherwise it gets very noisy. Logging overhaul is def a needed task but that's best left for a future ticket (ideally it's a flag and it just logs to a file).

Reply:
Ah yes - I added this just to look for my own purposes but forgot to remove! Thanks for the catch